cmake/external/gflags.cmake

```diff
@@ -14,8 +14,8 @@
 
 include(ExternalProject)
 
-set(GFLAGS_PREFIX_DIR ${THIRD_PARTY_PATH}/gflags)
 set(GFLAGS_INSTALL_DIR ${THIRD_PARTY_PATH}/install/gflags)
+set(GFLAGS_PREFIX_DIR ${THIRD_PARTY_PATH}/gflags)
```

Comment:
Why do you move this line?

Reply:
file name changes need to change cmake/external content,  to trigger remove third-party for recompile in ci process, this is sugges by ci oncall